### DISCUSSION TITLE
Missing generated types

### DISCUSSION THREAD

**Discussion by aweijun** [prisma/prisma#15964] [Q&A] [answered]
Hi I am currently using Prisma, I notice that I do not have access to some of the auto-generated types for certain models. For instance,
my User Model is missing GetPayLoad type but has access to the CountOutputTypeGetPayload. The weird thing is that this issue only occurs to certain models.

```
model User {
  id           String   @id(map: "PK__AD__3123") @db.UniqueIndentifier
  user       String db.NVarChar(500)
 @@map("TEST_USER_DETAILS")
}
```
Am I doing something wrong when creating the models?

Nevermind, this is related to the following issue https://github.com/prisma/prisma/issues/7014

**Comment by austincrim** [comment, marked as answer]
Hey @aweijun,

Were you able to resolve this by following [this solution](https://github.com/prisma/prisma/issues/7014#issuecomment-841255804)?

**Reply by awj99** [reply]
Yeap! Turns out to be a typescript version issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
